docs/usage.md

```diff
@@ -790,3 +793,41 @@ def nav_items
   ]
 end
 ```
+
+---
+
+# Nested Forms
+
+![Nested Table](images/nested_table.png)
+
+Fae provides an easy way to manage associated objects in one form via nested forms. A good use case for nested forms is when you have a `has_many` association where the child object is simple enough that an embedded table and form on the parent form will do. This makes managing the associated content much more convenient.
+
+Some good examples ideal for nested tables are: image galleries, tasting notes on a wine release and quotes on a person object.
+
+## Setting Up a Nested Form
+
+### Generate the child object
+
+Assuming the parent object already exists, generate the object to be nested with [Fae's nested scaffold command](#faenested_scaffold).
```

Comment:
That's the actual anchor tag: https://github.com/wearefine/fae/blob/master/docs/usage.md#faenested_scaffold